src/test/java/org/jenkinsci/plugins/workflow/EnvWorkflowTest.java

```diff
@@ -46,14 +46,15 @@
      */
     @Test public void isNodeNameAvailable() throws Exception {
         r.createSlave("node-test", "unix fast", null);
+        String builtInNodeLabel = r.jenkins.getSelfLabel().getExpression(); // compatibility with 2.307+
         WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "workflow-test");
 
         p.setDefinition(new CpsFlowDefinition(
-            "node('master') {\n" +
+            "node('" + builtInNodeLabel + "') {\n" +
```

Comment:
```suggestion
            "node('built-in || master') {\n" +
```

would also work right?

Reply:
Yes.